main.py

```diff
@@ -148,32 +153,25 @@ async def keen(context):
     await context.send(result)
 
 
-@bot.command(name='daily', help='Will give the daily leaderboard for specified data')
+@bot.command(name='daily', help='Will give the daily leaderboard for specified day')
 async def daily(context, day):
     # Only respond if used in a channel called 'advent-of-code'
     if context.channel.name != 'advent-of-code':
         return
 
     players = get_players()
 
-    players_day = []
-
     # Goes through all the players checking if they have data for that day and if they do adding to players_days
-    for player in players:
-        if day in player[4]:
-            players_day.append(player)
-            
-
+    players_day = [player for player in players if day in player[4]]
+ 
     #players_day has all people who have finished one star for that day
     first_star = []
     second_star = []
 
-    # Adds all the players which has 
+    # Adds all the players which has stars into respective lists
```

Comment:
"Adds all the players which have stars into the respective lists"

Reply:
Sorted thanks